packages/react-variant-view/src/util/ApiClientUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { GenomeNexusAPI } from 'genome-nexus-ts-api-client';
+import { OncoKbAPI } from 'oncokb-ts-api-client';
+
+// TODO customize domain?
+export const genomeNexusDomain = 'https://www.genomenexus.org/';
+const genomeNexusInternalClient = new GenomeNexusAPI(genomeNexusDomain);
```

Comment:
Right, most likely we would like to customize this. How do we do the genome nexus client customization for `react-mutation-mapper`? Is it possible to do it the same way?